docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,152 @@
+---
+description: How to install Docker EE for Windows
+keywords: windows, windows server, install, download
+title: Install Docker Enterprise Edition for Windows Server 2016
+---
+
+Docker Enterprise Edition for Windows Server 2016 (*Docker EE*) enables native
+Docker containers on Windows Server 2016. The Docker EE installation package
+includes everything you need to run Docker on Windows Server 2016.
+This topic describes pre-install considerations, and how to download and
+install Docker EE.
+
+> **Already have Docker EE for Windows?** If you already have Docker EE for
+Windows installed, and you're ready to get started, skip to
+[Get started with Docker for Windows](/docker-for-windows/index.md) for a quick
+tour of the command line, settings, and tools.
+>
+>**Looking for Release Notes?** [Get release notes for all
+versions here](https://docs.docker.com/release-notes/) or subscribe to the
+[releases feed on the Docker Blog](http://blog.docker.com/category/engineering/docker-releases/).
+
+## Install Docker EE
+
+Docker EE for Windows requires Windows Server 2016. See [What to know before
+you install](/docker-ee-for-windows/#what-to-know-before-you-install) for a
+full list of prerequisites. To install Docker Community Edition (*Docker CE*)
+on a Windows 10 machine, see [Install Docker for Windows](/docker-for-windows/install.md).
+
+1.  Open a PowerShell command prompt, and type the following commands.
+
+    ```ps
+    PS> Install-Module -Name DockerMsftProvider -Force
+    PS> Install-Package -Name docker -ProviderName DockerMsftProvider -Force
+    PS> Restart-Computer -Force
+    ```
+
+2.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```ps
+    PS> docker run hello-world:nanoserver
+
+    Unable to find image 'hello-world:nanoserver' locally
+    nanoserver: Pulling from library/hello-world
+    bce2fbc256ea: Pull complete
+    3ac17e2e6106: Pull complete
+    8cac44e17f16: Pull complete
+    5e160e4d8db3: Pull complete
+    Digest: sha256:25eac12ba40f7591969085ab3fb9772e8a4307553c14ea72d0e6f98b2c8ced9d
+    Status: Downloaded newer image for hello-world:nanoserver
+
+    Hello from Docker!
+    This message shows that your installation appears to be working correctly.
+    <snip>
+    ```
+
+
+## Using a script to install Docker EE
+
+Use the following steps when you want to install manually, script automated
+installs, or install on air-gapped systems.
+
+1.  In a PowerShell command prompt, download the installer archive on a machine
+that has a connection.
```

Comment:
nit: Please line 'that' up with 'In' in the future. Believe it or not, there are some times when this will fail to render properly and it's hard to debug.